dojo/tools/dependency_check/parser.py

```diff
@@ -238,8 +246,22 @@ def get_finding_from_vulnerability(self, dependency, related_dependency, vulnera
                                      'source: {1}\n' \
                                      'url: {2}\n\n'.format(name, source, url)
 
-        mitigation = 'Update ' + component_name + ':' + component_version + ' to at least the version recommended in the description'
-        description += '\nFilepath: ' + str(dependency_filepath)
+        if related_dependency is not None:
+            tags.append("related")
+
+        if vulnerability.tag == "{}suppressedVulnerability".format(namespace):
+            if notes == "":
+                notes = "Document on why we are suppressing this vulnerability is missing!"
+                tags.append("no_suppression_document")
+            mitigation = '**This vulnerability is mitigated and/or suppressed:** {}'.format(notes)
+
+            active = False
+            tags.append("suppressed")
+
+        else:
+            mitigation = 'Update {}:{} to at least the version recommended in the description'.format(component_name, component_version)
+            description += '\nFilepath: ' + str(dependency_filepath)
```

Comment:
I would add these lines even when the vulnerability is suppressed (after the mitigation line set above)

Reply:
resolved.